examples/guestbook-go/README.md

```diff
@@ -159,20 +159,34 @@ redis-master            <none>                                    name=redis,rol
 redis-slave             name=redis,role=slave                     name=redis,role=slave        10.0.22.180         6379
 ```
 
-To play with the service itself, find the external IP of the load balancer from the [Google Cloud Console][cloud-console] or the `gcloud` tool, and visit `http://<ip>:3000`.
-
-```shell
-$ gcloud compute forwarding-rules describe --region=us-central1 guestbook
-IPAddress: 11.22.33.44
-IPProtocol: TCP
-creationTimestamp: '2014-11-24T16:08:15.327-08:00'
-id: '17594840560849468061'
-kind: compute#forwardingRule
-name: guestbook
-portRange: 1-65535
-region: https://www.googleapis.com/compute/v1/projects/jbeda-prod/regions/us-central1
-selfLink: https://www.googleapis.com/compute/v1/projects/jbeda-prod/regions/us-central1/forwardingRules/guestbook
-target: https://www.googleapis.com/compute/v1/projects/jbeda-prod/regions/us-central1/targetPools/guestbook
+To play with the service itself, find the external IP of the load balancer as shown below by extracting a value for `publicIPs` (in this example `107.178.218.166`) and then visit `http://<ip>:3000`.
+
+```shell
+$ cluster/kubectl.sh get services guestbook -o json
```

Comment:
Optional: if you use a template as the output, you can extract just the publicIPs. Try
```
kubectl get services guestbook -o template --template='{{index . "publicIPs"}}'
```